ngraph/python/src/pyngraph/ops/constant.cpp

```diff
@@ -106,6 +106,18 @@ void regclass_pyngraph_op_Constant(py::module m)
                           const std::vector<uint64_t>&>());
 
     constant.def("get_value_strings", &ngraph::op::Constant::get_value_strings);
+    constant.def("cast_vector_char", &ngraph::op::Constant::cast_vector<char>);
+    constant.def("cast_vector_float", &ngraph::op::Constant::cast_vector<float>);
+    constant.def("cast_vector_double", &ngraph::op::Constant::cast_vector<double>);
+    constant.def("cast_vector_int8", &ngraph::op::Constant::cast_vector<int8_t>);
+    constant.def("cast_vector_int16", &ngraph::op::Constant::cast_vector<int16_t>);
+    constant.def("cast_vector_int32", &ngraph::op::Constant::cast_vector<int32_t>);
+    constant.def("cast_vector_int64", &ngraph::op::Constant::cast_vector<int64_t>);
+    constant.def("cast_vector_uint8", &ngraph::op::Constant::cast_vector<uint8_t>);
+    constant.def("cast_vector_uint16", &ngraph::op::Constant::cast_vector<uint16_t>);
+    constant.def("cast_vector_uint32", &ngraph::op::Constant::cast_vector<uint32_t>);
+    constant.def("cast_vector_uint64", &ngraph::op::Constant::cast_vector<uint64_t>);
+
```

Comment:
We need to find a better way to do this. One method such as `get_data` which performs the right cast internally. You're exposing C++ implementation details to Python.

Reply:
I've decided to use name `get_vector` instead. The name `get_data` is already reserved and return data in the same original shape, `get_vector` return flatten result.